pkg/unikontainers/hypervisors/vmm.go

```diff
@@ -66,6 +67,12 @@ func NewVMM(vmmType VmmType) (vmm VMM, err error) {
 			return nil, ErrVMMNotInstalled
 		}
 		return &Qemu{binary: QemuBinary, binaryPath: vmmPath}, nil
+	case FirecrackerVmm:
+		vmmPath, err := exec.LookPath(FirecrackerBinary + cpuArch())
```

Comment:
We should omit the `cpuArch()` part, as the `firecracker` binary is typically named `firecracker`.

Reply:
Yes, that is a good point. I have addressed this comment and pushed the changes.